apps/facix/package.json

```
@@ -30,5 +30,8 @@
     "config": "*",
     "tsconfig": "*",
     "typescript": "^4.5.5"
+  },
+  "engines": {
+    "node": "16"
```

Remove engines field cause it broken ci/cd
```suggestion
  },
  "engines": {
    "node": "16"
```